app/helpers/application_helper.rb

```diff
@@ -1,2 +1,13 @@
 module ApplicationHelper
+  MAX_VOTES = 3
+
+  private
+  def user_votes
+    (cookies["#{@hackday.id}_votes"] || MAX_VOTES).to_i
```

Comment:
i think putting this in application helper is bad form.

is it used across multiple controllers. if not it shouldnt be.

otherwise it should be in a concern.